packages/kbn-apm-synthtrace/src/scenarios/helpers/random_names.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const randomNames = [
```

Comment:
this should be renamed to ancient Greek random names 😆

Reply:
I thought you and @gbamparop would like it :D